spec/datetimeformat.html

```diff
@@ -588,7 +585,7 @@ <h1>Internal slots</h1>
       </ul>
 
       <p>
-        EXAMPLE     An implementation might include the following record as part of its English locale data: {[[hour]]: *"numeric"*, [[minute]]: *"2-digit"*, [[second]]: *"2-digit"*, [[pattern]]: *"{hour}:{minute}:{second}"*, [[pattern12]]: *"{hour}:{minute}:{second} {ampm}"*}.
+        <strong>EXAMPLE</strong> An implementation might include the following record as part of its English locale data: {[[hour]]: *"numeric"*, [[minute]]: *"2-digit"*, [[second]]: *"2-digit"*, [[pattern]]: *"{hour}:{minute}:{second}"*, [[pattern12]]: *"{hour}:{minute}:{second} {ampm}"*}.
```

Comment:
I think this and similar content would be better refactored with \<emu-note> to match ECMA 262 (e.g., https://tc39.es/ecma262/#sec-runtime-semantics-unicodematchproperty-p ).